### PR TITLE
ChallengeDetailsPage

### PR DESCRIPTION
**Pull request by okg00dbye**
TODO (list for myself haha):
- link checking off tasks
- fetch user tasks from be
- more styling
- challenge color

### CODE REVIEW

**Comment by jialin7878** [comment]
I CAN DO IT !!!

**Comment by jialin7878** [comment]
rebased to weim's branch

File: frontend/package.json [hunk 1]
```diff
@@ -15,6 +17,7 @@
     "axios": "^0.22.0",
     "browser-image-compression": "^1.0.16",
     "date-fns": "^2.25.0",
+    "dayjs": "^1.10.7",
```
Comment (sebastiantoh) [inline frontend/package.json:20]: (can disregard this comment for now)

is it necessary to use dayjs as well? I've already added `date-fns` which may have overlaps in terms of functionalities

**Review by sebastiantoh — APPROVED** [review]
lgtm!

**Pull request merged**